vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_unblinded.cc

```diff
@@ -346,13 +558,69 @@ void Unblinded::OnSendTokens(
 
 void Unblinded::ContributionCompleted(
     const ledger::Result result,
-    const std::string& viewing_id) {
-  const auto reconcile = ledger_->GetReconcileById(viewing_id);
-  ledger_->ReconcileComplete(
+    ledger::ContributionInfoPtr contribution) {
+  if (!contribution) {
+    return;
+  }
+
+  ledger_->ContributionCompleted(
       result,
-      reconcile.fee,
-      viewing_id,
-      reconcile.type);
+      contribution->amount,
+      contribution->contribution_id,
+      contribution->type);
+}
+
+void Unblinded::SetTimer(uint32_t* timer_id, uint64_t start_timer_in) {
```

Comment:
for the future better to use `const uint64_t&` as we still have x86 builds

Reply:
done